blog/index.html

```diff
@@ -590,7 +614,7 @@ <h2 class="entry-title"><a href="/2020/11/30/stackstorm-project-updates-autumn-2
 		<div class="wrp">
 			<div class="ft-c">
 				<p>
-											© 2021 StackStorm a Series of LF Projects, LLC. All rights reserved. For web site terms of use, trademark policy and other project policies please see <a href="https://lfprojects.org/">https://lfprojects.org/</a>. 
+											© 2021 StackStorm a Series of LF Projects, LLC. All rights reserved. For web site terms of use, trademark policy and other project policies please see <a href="https://lfprojects.org/">https://lfprojects.org/</a>. 
```

Comment:
2022? 2023?

Reply:
Would like to avoid changing a single page here to 2022 and still have 2021 everywhere else in the footer. 
Perhaps it makes sense when we'll have a templating engine in place.